presto-thrift-connector/src/main/java/com/facebook/presto/connector/thrift/util/RetryDriver.java

```diff
@@ -125,15 +191,23 @@ public RetryDriver withClassifier(Function<Exception, Exception> classifier)
 
         while (true) {
             retryStatus.nextAttempt();
+            getStats(callableName, stats -> stats.updateAttempt());
```

Comment:
Let's not add the number of total attempts for now as it's possible to get it as a sum of the other two.